regression-test/suites/variant_p0/desc.groovy

```diff
@@ -0,0 +1,194 @@
+            set 'format', 'json' 
+            set 'max_filter_ratio', '0.1'
+            file file_name // import json file
+            time 10000 // limit inflight 10s
+
+            // if declared a check callback, the default check condition will ignore.
+            // So you must check all condition
+
+            check { result, exception, startTime, endTime ->
+                if (exception != null) {
+                        throw exception
+                }
+                logger.info("Stream load ${file_name} result: ${result}".toString())
+                def json = parseJson(result)
+                assertEquals("success", json.Status.toLowerCase())
+                // assertEquals(json.NumberTotalRows, json.NumberLoadedRows + json.NumberUnselectedRows)
+                assertTrue(json.NumberLoadedRows > 0 && json.LoadBytes > 0)
+            }
+        }
+    }
+
+    def create_table = { table_name, buckets="auto" ->
+        sql "DROP TABLE IF EXISTS ${table_name}"
+        sql """
+            CREATE TABLE IF NOT EXISTS ${table_name} (
+                k bigint,
+                v variant
+            )
+            DUPLICATE KEY(`k`)
+            DISTRIBUTED BY HASH(k) BUCKETS ${buckets}
+            properties("replication_num" = "1", "disable_auto_compaction" = "false");
+        """
+    }
+
+    def create_table_partition = { table_name, buckets="auto" ->
+        sql "DROP TABLE IF EXISTS ${table_name}"
+        sql """
+            CREATE TABLE IF NOT EXISTS ${table_name} (
+                k bigint,
+                v variant
+            )
+            DUPLICATE KEY(`k`)
+            PARTITION BY RANGE(k)
+            (
+                PARTITION p1 VALUES LESS THAN (3000),
+                PARTITION p2 VALUES LESS THAN (50000),
+                PARTITION p3 VALUES LESS THAN (100000)
+            )
+            DISTRIBUTED BY HASH(k) BUCKETS ${buckets}
+            properties("replication_num" = "1", "disable_auto_compaction" = "false");
+        """
+    }
+
+    def set_be_config = { key, value ->
+        String backend_id;
+        def backendId_to_backendIP = [:]
+        def backendId_to_backendHttpPort = [:]
+        getBackendIpHttpPort(backendId_to_backendIP, backendId_to_backendHttpPort);
+
+        backend_id = backendId_to_backendIP.keySet()[0]
+        def (code, out, err) = update_be_config(backendId_to_backendIP.get(backend_id), backendId_to_backendHttpPort.get(backend_id), key, value)
+        logger.info("update config: code=" + code + ", out=" + out + ", err=" + err)
+    }
+
+    try {
+        // sparse columns
+        def table_name = "sparse_columns"
+        create_table table_name
+        set_be_config.call("variant_ratio_of_defaults_as_sparse_column", "0.95")
+        sql """set describe_extend_variant_column = true"""
+        sql """insert into  sparse_columns select 0, '{"a": 11245, "b" : [123, {"xx" : 1}], "c" : {"c" : 456, "d" : null, "e" : 7.111}}'  as json_str
```

Comment:
add chinese column name and unicode column, to test if could be output well

Reply:
done